exercises/pangram/canonical-data.json

```diff
@@ -3,66 +3,78 @@
   "comments": [
     "A pangram is a sentence using every letter of the alphabet at least once."
   ],
-  "version": "1.1.0",
+  "version": "1.2.0",
   "cases": [
     {
-      "description": "Check if the given string is an pangram",
+      "description": "checks if the given string is a pangram.",
       "comments": [
         "Output should be a boolean denoting if the string is a pangram or not."
       ],
       "cases": [
         {
-          "description": "sentence empty",
+          "description": "returns false for an empty message.",
           "property": "isPangram",
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "returns false for an undefined/null message as argument.",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": null,
```

Comment:
If `null` is tested as an input, I am thinking it should be an error, rather than to return false. The reason for this belief is that if we consider that the function under test must only accept strings, well `null` is not a string, so this is in error.

(Of course, I imagine languages for which strings are non-nullable will simply skip this test)

This comment must not be read as an endorsement or rejection of the statement "`null` should be tested as an input".

Reply:
Yeah, I agree. I had this same thought but had not changed it, yet, because of the ongoing discussion. I will probably change it to expect a throw in an upcoming commit.